code/client/munkilib/prefs.py

```diff
@@ -89,6 +89,7 @@
     'SuppressLoginwindowInstall': False,
     'SuppressStopButtonOnInstall': False,
     'SuppressUserNotification': False,
+    'SuspendDisplaySleepDuringBootstrap': False,
```

Comment:
I'm wondering if rather than Yet Another Preference for admins to discover and implement, if it might not make more sense to just make this the normal behavior, _but only during actual bootstrapping_, and _only if we're on AC power or have more than XX% battery (maybe 50% on Intel and 30% on Apple silicon?)

Reply:
I agree that sometimes the last thing we need is another preferences. I could see leaving the display on for a bootstrap run be the default, with those defaults for a battery run. With, maybe a preference to ignore if on battery — I could possible see that being of some interest.